.env

```diff
@@ -2,4 +2,5 @@ PORT=8083
 EXTRA_PORTS='8082'
 WEBPACK_DEV_URL=http://0.0.0.0:8082
 BASE_URL=http://0.0.0.0:8083
-OPENID_VERIFY_URL=http://0.0.0.0:8083/login/verify
\ No newline at end of file
+OPENID_VERIFY_URL=http://0.0.0.0:8083/login/verify
+SNAPCRAFT_URL=https://snapcraft.io
```

Comment:
I think this `.env` is only used when running with run script. Would be good to add `SNAPCRAFT_URL` to `src/config/defaults.js` as well, to make sure some default value is set.

Reply:
Done, thanks. Didn't know that was the case :+1: